.github/workflows/cypress-workflow.yml

```diff
@@ -7,8 +7,8 @@ on:
     branches:
       - "*"
 env:
-  OPENSEARCH_DASHBOARDS_VERSION: '2.4.0'
-  OPENSEARCH_VERSION: '2.4.0-SNAPSHOT'
+  OPENSEARCH_DASHBOARDS_VERSION: '2.4.1'
```

Comment:
@gaiksaya  Sorry I didn't notice this before. the `2.4.1` here attempts to access that `tag` in the OpenSearch-Dashboards repo, which doesn't exist yet. Could you change this to `2.4`?

Reply:
@gaiksaya I think the cypress tests are failing because the `OPENSEARCH_DASHBOARDS_VERSION` needs to be `2.4`.